app/forms/api/profile_creation_form.rb

```diff
@@ -127,7 +139,7 @@ def form_valid?
     def extra_attributes
       if user.present?
         @extra_attributes ||= {
-          profile_pending: user.pending_profile?,
+          profile_pending: user.pending_profile? && user_bundle.gpo_address_verification?,
```

Comment:
Yeah, in earlier iterations I had tried to avoid adding a new deactivation reason and reusing the existing `verification_pending`, disambiguating based on the profile address verification method. However, this couldn't account for a GPO + IPP profile, where we'd have an issue that we'd need to be able to mark the profile as GPO-complete + IPP-incomplete, something which didn't seem possible without the new deactivation reason.

So technically this change might not be necessary any longer. I could go either way about keeping it, since technically the other changes here sorta lean into the idea that "pending profile" = "gpo". On the other hand, there is some other existing code that this helps aligns toward:

https://github.com/18F/identity-idp/blob/b25790b34d3c542f00849808dc37e8d0a7ec64e7/app/controllers/idv/personal_key_controller.rb#L36-L37